File: iOS8_Adv_Training/Localizing/Localizing/ViewController.m
//
//  ViewController.m
//  Localizing
//
//  Created by 新居雅行 on 2014/10/28.
//  Copyright (c) 2014年 msyk.net. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UITextField *input;
@property (weak, nonatomic) IBOutlet UITextView *message;
- (IBAction)tapButton:(id)sender;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)tapButton:(id)sender {
    NSString *seedString = NSLocalizedString(@"message1", nil);;
    NSString *msgString = [NSString stringWithFormat: seedString, self.input.text];
    self.message.text = msgString;
}
@end

File: iOS9_Adv_Training/Localizing_Swift/Localizing_Swift/ViewController.swift
//
//  ViewController.swift
//  Localizing_Swift
//
//  Created by 新居雅行 on 2015/01/29.
//  Copyright (c) 2015年 msyk.net. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet private var input: UITextField?
    @IBOutlet private var message: UITextView?
    
@IBAction func tapButton(sender: UIButton) {
    if let inputText = input?.text {
        message?.text = NSString(format:
            NSLocalizedString("message1", comment: ""), inputText) as String
    }
}
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    
}

